account_reconcile_oca/views/account_bank_statement_line.xml

```diff
@@ -349,6 +349,26 @@
             </p>
         </field>
     </record>
+    <record id="action_bank_statement_line_open" model="ir.actions.act_window">
+        <field name="name">Statement lines</field>
+        <field name="res_model">account.bank.statement.line</field>
+        <field name="domain">[('journal_id', '=', active_id)]</field>
+        <field
+            name="context"
+        >{'default_journal_id': active_id, 'view_ref': 'account_reconcile_oca.bank_statement_line_form_reconcile_view'}</field>
+        <field name="view_mode">kanban,tree</field>
+        <field
+            name="view_ids"
+            eval="[(5, 0, 0),
+            (0, 0, {'view_mode': 'kanban', 'view_id': ref('bank_statement_line_reconcile_view')}),
+            (0, 0, {'view_mode': 'tree', 'view_id': ref('account_statement_base.account_bank_statement_line_tree')})]"
+        />
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+            Nothing to reconcile
+            </p>
+        </field>
+    </record>
```

Comment:
`account_reconcile_oca.action_bank_statement_line_reconcile_all` is the same action
```suggestion
```